widgets/contact/js/contact.js

```diff
@@ -0,0 +1,61 @@
+var SiteOriginContactForm = {
```

Comment:
@braamgenis For the most part, we've been doing indenting using 4 spaces in our Javascript files. This file seems to be using 2 space indenting. Although we really should be using the WordPress [JS best practices](https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/) and using tab indents.

At some point we should do a more thorough run through all our code to make sure we're following these guidelines.